### PR TITLE
WEL-17

### PR DESCRIPTION
**Pull request by eugenchen**
Added footer

### CODE REVIEW

**Comment by YuriiSkyba1** [comment]
please create branches in format like WEL-17-footer with ticket title

**Comment by YuriiSkyba1** [comment]
Aslo we can use this commits format [WEL-17]: implemented footer component

[WEL-17]: https://vladysllav.atlassian.net/browse/WEL-17?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: src/components/Footer/facebook.svg
Comment (YuriiSkyba1) [inline src/components/Footer/facebook.svg:1]: please move all images to assets folder /images

File: src/components/Footer/instagram.svg
Comment (YuriiSkyba1) [inline src/components/Footer/instagram.svg:1]: please move all images to assets folder /images

File: src/components/Footer/logo.png
Comment (YuriiSkyba1) [inline src/components/Footer/logo.png:1]: please move all images to assets folder /images

File: src/components/Footer/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      <ul id="footer-rules">
+        <h1 className="footer-header">Rules</h1>
+        <li className="footer-nav-item">
+          <a href="/privacyPolicy">Privacy policy</a>
+        </li>
+        <li className="footer-nav-item">
+          <a href="/terms">Terms of service</a>
+        </li>
+      </ul>
+      <ul id="footer-links">
```
Comment (YuriiSkyba1) [inline src/components/Footer/footer.tsx]: You can split this to smaller components for example section nav menu and contacts and import these components to new Page/Footer

File: src/components/Footer/footer.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
+.footer-menu {
```
Comment (YuriiSkyba1) [inline src/components/Footer/footer.css]: you can use tailwind classes for styiling

File: src/components/Footer/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        </li>
+        <li className="footer-nav-item">
+          <a href="/profile">Profile</a>
+        </li>
+        <li className="footer-nav-item">
+          <a href="/services">Services</a>
+        </li>
+      </ul>
+
+      <ul id="footer-rules">
```
Comment (YuriiSkyba1) [inline src/components/Footer/footer.tsx]: using id it's bad approach please use classes instead id

**Review by YuriiSkyba1 — CHANGES_REQUESTED** [review]

**Comment by dadclak** [comment]
> please create branches in format like WEL-17-footer with ticket title

@YuriiSkyba1 do we must add prefix 'feature' -> `feature/WEL-17` ?

**Review by YuriiSkyba1 — APPROVED** [review]

**Review by YuriiSkyba1 — APPROVED** [review]

**Pull request merged**